packages/expo/src/singleton.ts

```diff
@@ -11,29 +13,56 @@ Clerk.sdkMetadata = {
 
 const KEY = '__clerk_client_jwt';
 
+/**
+ * @deprecated Use `getClerkInstance` instead. `Clerk` will be removed in the next major version.
```

Comment:
🙃 typo, `clerk`  should be with lower case to match the actual export.

Reply:
So i looked at this again, turns out the exported variable from the package is `Clerk` so the initial comment is correct